filecoin-proofs/src/api.rs

```diff
@@ -175,6 +227,22 @@ pub fn seal<T: Into<PathBuf> + AsRef<Path>>(
     let comm_d = commitment_from_fr::<Bls12>(public_tau.comm_d.into());
     let comm_r_star = commitment_from_fr::<Bls12>(tau.comm_r_star.into());
 
+    let valid_pieces = PieceInclusionProof::verify_all(
+        &comm_d,
+        &piece_inclusion_proofs,
+        &comm_ps,
+        &piece_specs
+            .into_iter()
+            .map(|p| p.length)
+            .collect::<Vec<_>>(),
+        (sector_bytes / 127) * 4,
+    )
+    .expect("pip verification sanity check failed");
```

Comment:
TODO: Drop this `expect`, as it panics. Short circuit using `?` instead. We'll do the same thing with the other `expect`/`expects` in this function in a subsequent PR.